src/components/core/Alert.vue

```diff
@@ -43,43 +42,35 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import { mdiClose } from '@mdi/js'
 import { mapActions, mapState } from 'vuex'
 
+// TODO: remove later when https://github.com/vuetifyjs/vuetify/issues/11021 is fixed
+const colors = new Map([
+  ['error', 'red'],
+  ['success', 'green'],
+  ['warning', 'amber']
+])
```

Comment:
This didn't need to be in `data` either, nor in options because it's not accessed in the template (only a computed property), so I've moved it out of the default object altogether